packages/shared_preferences/shared_preferences/README.md

```diff
@@ -50,9 +50,22 @@ latest data stored on the native platform regardless of what process was used to
 
 ### Android platform storage
 
-The [SharedPreferences] API uses the native [Android Shared Preferences](https://developer.android.com/reference/android/content/SharedPreferences) tool to store data.
+The [SharedPreferencesAsync] and [SharedPreferencesWithCache] APIs can use [DataStore Preferences](https://developer.android.com/topic/libraries/architecture/datastore) or [Android SharedPreferences](https://developer.android.com/reference/android/content/SharedPreferences) to store data.
+In most cases you should use the default option of DataStore Preferences, as it is the platform-recommended preferences storage system. 
+However, in some cases you may need to interact with preferences that were written to SharedPreferences by code you don't control.
 
-The [SharedPreferencesAsync] and [SharedPreferencesWithCache] APIs use [DataStore Preferences](https://developer.android.com/topic/libraries/architecture/datastore) to store data.
+To use the `Android SharedPreferences` backend, use the `SharedPreferencesAsyncAndroidOptions` when using [SharedPreferencesAsync] on Android.
+
+<?code-excerpt "readme_excerpts.dart (Android_Options)"?>
+```dart
```

Comment:
This excerpt should include the `import 'package:shared_preferences_android/shared_preferences_android.dart';` from line 7.